Adyen/Model/BrowserInfo.swift

```diff
@@ -25,7 +25,9 @@ public struct BrowserInfo: Encodable {
         webView = WKWebView()
         webView?.evaluateJavaScript("navigator.userAgent") { result, _ in
             webView = nil
-            guard let result = result as? String else { completion(nil); return }
+            guard let result = result as? String else {
+                return completion(nil)
+            }
```

Comment:
why?, one line is not long enough to make multiline

Reply:
my main focuse was `completion(nil); return` ..
I can do 

``` guard let result = result as? String else { return completion(nil) } ```